helm_deploy/laa-crown-court-proceeding/templates/_helpers.tpl

```diff
@@ -61,3 +61,14 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Create ingress configuration
+*/}}
+{{- define "laa-crown-court-proceeding.ingress" -}}
+{{- $internalAllowlistSourceRange := (lookup "v1" "Secret" .Release.Namespace "ingress-internal-allowlist-source-range").data.INTERNAL_ALLOWLIST_SOURCE_RANGE | b64dec }}
+{{- if $internalAllowlistSourceRange }}
+  nginx.ingress.kubernetes.io/whitelist-source-range: {{ $internalAllowlistSourceRange }}
+  external-dns.alpha.kubernetes.io/set-identifier: {{ include "laa-crown-court-proceeding.fullname" . }}-{{ $.Values.ingress.environmentName}}-green
```

Comment:
I noticed none of the other config in here is indented, just wondering if the fact that this is might cause issues when rendered into the ingress.yaml file, have you checked?

Reply:
Awesome 👍